### PR TITLE
modified transaction complete popup

### LINKED ISSUE #15759: [Bug]: Transaction complete popup shows etherscan text on any chain transaction
### Describe the bug

Made a transaction on polygon testnet and got this popup after the completion of transaction
image

Although it redirected me to mumbai.polygonscan but the text shows etherscan

### Steps to reproduce

1. Make a transaction on any chain except ethereum
2. You'll get a popup after the completion of transaction

### Version

10.18.3

### Browser

Chrome, Firefox, Brave

### Operating system

Windows

### Additional context

I would love to work on this issue if assigned:)

### PR DESCRIPTION
**Pull request by adnansahovic**
## Explanation

Changed the text on popup after the completion of transaction

## More Information

* Fixes #15759

### Before
188907813-0c312e2a-10ca-4d1e-95da-d1c6b9a90269

### After
Screenshot 2022-10-28 at 16 05 10

## Manual Testing Steps

1. Make a transaction on any chain except ethereum
2. You'll get a popup after the completion of transaction

### CODE REVIEW

**Comment by mirjanaKukic** [comment]
Verified by QA

**Review by brad-decker — DISMISSED** [review]

**Comment by brad-decker** [comment]
@adnansahovic needs a rebase.

File: app/scripts/platforms/extension.js [hunk 1]
```diff
@@ -160,10 +162,11 @@ export default class ExtensionPlatform {
 
     const url = getBlockExplorerLink(txMeta, rpcPrefs);
     const nonce = parseInt(txMeta.txParams.nonce, 16);
+    const view = getURLHostName(url).replace(/([.]\w+)$/u, '');
```
Comment (jpuri) [inline app/scripts/platforms/extension.js:165]: - Is this way of deriving view name from URL good enough ?
- `startCase(toLower(view))` this conversion can also be probably done here.

File: app/scripts/platforms/extension.js [hunk 2]
```diff
@@ -160,10 +162,13 @@ export default class ExtensionPlatform {
 
     const url = getBlockExplorerLink(txMeta, rpcPrefs);
     const nonce = parseInt(txMeta.txParams.nonce, 16);
+    const view = startCase(
+      toLower(getURLHostName(url).replace(/([.]\w+)$/u, '')),
```
Comment (brad-decker) [inline app/scripts/platforms/extension.js:166]: I have to agree with @jpuri on this one. Your screenshot for Mumbai PolygonScan makes it appear that this works with subdomains (mumbai) but then Ehterscan would have a 'www' proceeding it? And if I do a naive assertion on this regex statement in console: 

```
const y = new URL('https://mumbai.polygonscan.com/');
> undefined
y.hostname.replace(/([.]\w+)$/u, '')
> 'mumbai.polygonscan'
y.hostname.replace(/([.]\w+)$/ug, '')
> 'mumbai.polygonscan'
```

**Review by brad-decker — CHANGES_REQUESTED** [review]

File: app/scripts/platforms/extension.js (hunk 2 above)
Comment (brad-decker) [reply app/scripts/platforms/extension.js:166]: I missed the call to 'startCase' which is the reason the magic works. Alright i'm good with this. Thanks for pushing back @adnansahovic

**Review by brad-decker — APPROVED** [review]

**Review by jpuri — APPROVED** [review]

**Comment by darkwing** [comment]
Merging as there are two approvals!

**Pull request merged**